calico-cloud_versioned_docs/version-3.17/release-notes/index.mdx

```diff
@@ -4,6 +4,44 @@ description: What's new, and why features provide value for upgrading.
 
 # Release notes
 
+## August 8, 2023
```

Comment:
This needs to be updated to August 14, 2023

Reply:
Fixed.